package.json

```diff
@@ -47,7 +47,6 @@
         "typescript": "^3.9.6"
     },
     "dependencies": {
-        "@microsoft/powerquery-parser": "0.2.18",
```

Comment:
Removing the parser dependency?

Reply:
Sorry, I should've marked this as a draft. My goal was to queue up a PR for this and the formatter, where the formatter needs to be merged first (and `npm publish`'ed). In the meantime I use `npm link` which it doesn't include symbolic linked projects in `package.json`.